modules/Registration/containers/teacher/TeacherRegistrationContainer.tsx

```diff
@@ -19,14 +25,16 @@ const TeacherRegistrationContainer: React.FC = () => {
     };
     try {
       const result = await registerTeacher(registrationData).unwrap();
+      dispatch(setUser(result.user));
+      dispatch(setToken(result.token));
```

Comment:
Since you're storing the token in local storage, I don't see why you'd have to do it here as well.

Reply:
Bhaiya, I have already updated this on the next PR